studio/components/interfaces/Settings/General/Infrastructure/ProjectUpgradeAlert/ProjectUpgradeAlert.tsx

```diff
@@ -83,44 +87,89 @@ const ProjectUpgradeAlert = () => {
                         All services, including Auth, Rest, and Extensions will be upgraded. This
                         action cannot be undone.
                       </p>
+                      <Alert_Shadcn_ title="Your project will be offline while the upgrade is in progress">
+                        <IconAlertCircle className="h-4 w-4" strokeWidth={2} />
+                        <AlertTitle_Shadcn_>
+                          Your project will be offline while the upgrade is in progress
```

Comment:
front load the important information

```suggestion
                          Your project will be offline for up to {durationEstimateHours} hour{durationEstimateHours === 1 ? '' : 's'}
```